target_postgres/singer_stream.py

```diff
@@ -71,7 +71,11 @@ def update_schema(self, schema, key_properties):
 
         # The validator can handle _many_ more things than our simplified schema, and is, in general handled by third party code
         self.validator = Draft4Validator(schema, format_checker=FormatChecker())
-
+        
+        # Some taps do not provide 'properties' key, which causes the next block to fail
+        if not (self.schema and 'properties' in self.schema):
+            return
+        
```

Comment:
Wouldn't it be easier to write something like this below?
`properties = self.schema.get('properties', {})
`
among others to ensure that the test on `self.key_properties` is executed? (not sure if that situation makes sense logically though)

Reply:
I believe that should work just as well for my case, I'll update it tonight.

Thanks @laurentS